site/src/components/message.md

```diff
@@ -76,4 +104,32 @@ Use the `message-sender` class to indicate message sent by the user.
 		</p>
 	</div>
 </article>
+<div class="margin-sm"></div>
+<article class="message message-sender message-size-sm">
+	<p class="message-time">May 29, 2023, 4:55PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="message-content-header">
+			<div class="persona">
+				<div class="persona-details">
+					<p class="persona-name">Kay Smith</p>
+				</div>
+			</div>
+			<details class="message-content-options popover popover-right">
+				<summary class="button button-clear">⋮</summary>
+				<div class="popover-content">Popover content.</div>
+			</details>
+		</div>
+		<p>
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Viverra aliquet eget sit amet tellus cras. Nisl nisi
+			scelerisque eu ultrices vitae. Condimentum id venenatis a condimentum. Ut eu sem integer
+			vitae.
+		</p>
+	</div>
+</article>
 ```
```

Comment:
may want to add section for text sizes like in fonts

Reply:
I added a separate section 'Sizes'